core/services/relay/evm/mercury/wsrpc/client.go

```diff
@@ -307,6 +443,14 @@ func (w *client) LatestReport(ctx context.Context, req *pb.LatestReportRequest)
 	if err = w.waitForReady(ctx); err != nil {
 		return nil, errors.Wrap(err, "LatestReport failed")
 	}
+
+	if w.tlsCertFile != nil {
+		signature, err := w.Sign(req)
```

Comment:
Is there any defense against a replay attack? Besides TLS, how do we prevent someone from sending the same payload again, or to other nodes?

Is replay attack a concern?

Reply:
A replay attack is not a concern - the server has logic to discard processing of non-unique reports. 

In this case, the node is talking to 1...N web servers (all mercury servers).